app/javascript/elm/src/Main.elm

```diff
@@ -59,6 +59,8 @@ type alias Model =
     , resetIcon : String
     , heatMapThresholds : WebData HeatMapThresholds
     , isStreaming : Bool
+    , isSearchOn : Bool
```

Comment:
Have you considered using the full feature name? That means `isSearchAsIMoveOn`. It's a bit confusing with just `isSearchOn`

Reply:
or `isSearchAsIMove`